charts/dapr/charts/dapr_config/templates/dapr_default_config.yaml

```diff
@@ -9,6 +9,14 @@ metadata:
     {{ $key }}: {{ tpl $value $ }}
     {{- end }}
 spec:
+  metrics:
+    enabled: {{ .Values.global.prometheus.enabled }}
```

Comment:
I don't like the idea of using `{{ .Values.global.prometheus.enabled }}` to indicate whether or not metrics should be enabled. I suppose that could be an independent value, but it seems strange that you could potentially have `metrics.enabled: false` and `prometheus.enabled: true`.